src/postgraphile/http/createPostGraphileHttpRequestHandler.js

```diff
@@ -544,8 +544,14 @@ export default function createPostGraphileHttpRequestHandler(options) {
       if (result && result.errors)
         result.errors = result.errors.map(formatError)
 
-      res.setHeader('Content-Type', 'application/json; charset=utf-8')
-      res.end(JSON.stringify(result))
+      if (options.handleErrors && result.errors) {
+        result.errors = options.handleErrors(result.errors, req, res)
+      }
```

Comment:
I'd prefer that `handleErrors` *replaced* `formatError` so that it has access to the GraphQLError directly - once it's gone through `formatError` some content is lost. So something more like: